go/pfconfigdriver/pool.go

```diff
@@ -37,7 +37,7 @@ func NewPool() Pool {
 	p := Pool{}
 	p.lock = timedlock.NewRWLock()
 	p.lock.Timeout = 1 * time.Second
-	p.lock.RTimeout = 10 * time.Second
+	p.lock.RTimeout = 2 * time.Minute
```

Comment:
No, it's because there can be longer running requests now through the API frontend and this ensures the pfconfigdriver pool can wait a long enough time